package.json

```diff
@@ -54,6 +54,7 @@
     "passport-jwt": "^4.0.0",
     "passport-local": "~0.1.6",
     "passport-twitter": "^1.0.4",
+    "request": "^2.87.0",
```

Comment:
Hi @ddouglasz I think `supertest` does what this package does for you. Why do you have it installed?

Reply:
Thank you @fortunee. I have fixed it. I took it off.